config/crd/bases/api-gateway.consul.hashicorp.com_meshservices.yaml

```diff
@@ -0,0 +1,52 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.0
+  creationTimestamp: null
+  name: meshservices.api-gateway.consul.hashicorp.com
+spec:
+  group: api-gateway.consul.hashicorp.com
+  names:
+    kind: MeshService
+    listKind: MeshServiceList
+    plural: meshservices
+    singular: meshservice
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: MeshService holds a reference to an externally managed Consul
+          Service Mesh service.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
```suggestion
              internal value and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Reply:
that's an auto-generated comment, so you'll need to take it up with the kubebuilder folks upstream 😁